databases/redis/src/opnsense/mvc/app/models/OPNsense/Redis/Redis.xml

```diff
@@ -0,0 +1,109 @@
+<model>
+  <mount>//OPNsense/redis</mount>
+  <description>Redis DB</description>
+  <items>
+    <general>
+      <enabled type="BooleanField">
+        <default>0</default>
+        <Required>Y</Required>
+      </enabled>
+      <listen type="InterfaceField">
+        <default>1</default>
+        <Required>N</Required>
+        <multiple>Y</multiple>
+      </listen>
+      <protected_mode type="BooleanField">
+        <default>1</default>
+        <Required>Y</Required>
+      </protected_mode>
+      <port type="IntegerField">
+        <MinimumValue>0</MinimumValue>
```

Comment:
What actually happens on 0?

Reply:
0 means disabled port (no port binding).

Reply:
Unix socket mode only? Do we want this?

Reply:
maybe yes (if no application uses the TCP Port, there is no reason to have it open).